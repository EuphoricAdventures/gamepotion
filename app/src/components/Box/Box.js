import React from 'react'
import styled from 'styled-components'

const StyledBox = styled.div`
  display: block;
  padding: 2rem;
  border-radius: 4px;
  box-shadow: 0 4px 20px rgb(212, 212, 212);
  background-color: #f0f3f4;
`

const Box = ({ children }) => {
  return (
    <StyledBox className='component--box'>
      {children}
    </StyledBox>
  )
}

export default Box
