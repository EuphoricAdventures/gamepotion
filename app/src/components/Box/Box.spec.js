import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Box from './Box'

it('renders', () => {
  const tree = renderer
    .create(
      <Box>Hello, World</Box>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
