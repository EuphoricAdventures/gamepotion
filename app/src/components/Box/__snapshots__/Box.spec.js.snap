// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  display: block;
  padding: 2rem;
  border-radius: 4px;
  box-shadow: 0 4px 20px rgb(212,212,212);
  background-color: #f0f3f4;
}

<div
  className="component--box c0"
>
  Hello, World
</div>
`;
